kolibri/plugins/learn/assets/src/views/ExploreLibrariesPage/index.vue

```diff
@@ -92,24 +93,57 @@
     },
     mixins: [commonCoreStrings, commonLearnStrings],
     setup() {
+      const { networkDevices } = useDevices();
       const { fetchChannels } = useChannels();
-      const { back } = useContentLink();
-      const { fetchDevices } = useDevices();
       const { createPinForUser, deletePinForUser, fetchPinsForUser } = usePinnedDevices();
+      const { back } = useContentLink();
+      const deviceChannelsMap = reactive({});
+      const loading = ref(false);
```

Comment:
I think we should set this to start as `true` for initial page load, then it just gets set false at the first time we have a reason to stop showing the loading state.